### PR TITLE
Add more detail to sample

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,36 +8,50 @@ Run `dart run wasm:setup` to build the Wasmer runtime.
...
+      final mod = WasmModule(data);
+      print(mod.describe());
+      final inst = mod.builder().build();
+      final square = inst.lookupFunction('square');
+      print(square(12));
+    }
+    ```
+
+1. Copy wasmer runtime built during setup
+(`/wasm/.dart_tool/wasm/libwasmer.dylib`; `/wasmer.dll` on Windows)
```
Comment (kevmoo) [inline README.md:51]: indent here is not consistent. 😄

**Review by kevmoo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mit-mit) [reply README.md:51]: This wasn't meant to be indented; it's a non-semantic linebreak :-)

Comment (mit-mit) [reply README.md:51]: It looks like this renders as expected:

https://github.com/dart-lang/wasm/blob/e19dd2c855004c77406f06256166dc8f613bd499/README.md

File: README.md [hunk 2]
```diff
@@ -8,36 +8,50 @@ Run `dart run wasm:setup` to build the Wasmer runtime.
...
+      final data = File('square.wasm').readAsBytesSync();
+      final mod = WasmModule(data);
+      print(mod.describe());
+      final inst = mod.builder().build();
+      final square = inst.lookupFunction('square');
+      print(square(12));
+    }
+    ```
+
+1. Copy wasmer runtime built during setup
```
Comment (liamappelbe) [inline README.md:50]: You shouldn't have to manually copy files around. I think I fixed this bug. If you still have to do this, can you file a bug?

Comment (mit-mit) [reply README.md:50]: Filed https://github.com/dart-lang/wasm/issues/33

Comment (mit-mit) [reply README.md]: And removed it from this PR for now

**Review by liamappelbe — APPROVED** [review]

**Pull request merged**